ui/org.eclipse.pde.core/src/org/eclipse/pde/internal/core/product/RepositoryInfo.java

```diff
@@ -78,6 +95,7 @@ public void write(String indent, PrintWriter writer) {
 		if (isURLDefined()) {
 			writer.print(indent + "<repository location=\"" + fURL + "\""); //$NON-NLS-1$ //$NON-NLS-2$
 			writer.print(" enabled=\"" + fEnabled + "\""); //$NON-NLS-1$//$NON-NLS-2$
+			writer.print(" name=\"" + fName + "\""); //$NON-NLS-1$//$NON-NLS-2$
```

Comment:
Can you please move the name as second Argument before enabled?
The Name is IMO of higher interest since most of the time enabled is true any ways.